examples/README.md

```diff
@@ -211,6 +212,13 @@ Example | File | Description
 `shader_custom_material` | [`shader/shader_custom_material.rs`](./shader/shader_custom_material.rs) | Illustrates creating a custom material and a shader that uses it
 `shader_defs` | [`shader/shader_defs.rs`](./shader/shader_defs.rs) | Demonstrates creating a custom material that uses "shaders defs" (a tool to selectively toggle parts of a shader)
 
+## Tasks
```

Comment:
Maybe "Tasks" is not the most evocative category name. "Parallel Tasks"? "Async Tasks"?

Reply:
I like `Async Tasks` as well.

Reply:
My reasoning for choosing Tasks was that, for the most part, example sections translate directly to Bevy crate names and module exports, i.e. input -> bevy_input, audio -> bevy_audio, etc. I thought it would be best to maintain existing consistency between crate names, module exports, and example categories.

Reply:
What do you guys thinks? @mockersf @NathanSWard

Reply:
I think "Tasks" is not very discoverable, but "async" is in the name of the example so it should still be picked up by someone searching the keyword in the page. So... no strong opinion on my part here

Reply:
I share the same opinion about as @mockersf

Reply:
``Async Tasks`` it is 👍

Reply:
Changed in d1c7582